shell/platform/darwin/common/BUILD.gn

```diff
@@ -16,15 +16,7 @@ source_set("common") {
     "command_line.mm",
   ]
 
-  deps = [
-    "//flutter/common",
-    "//flutter/flow",
-    "//flutter/fml",
-    "//flutter/runtime",
-    "//flutter/shell/common",
-    "//third_party/dart/runtime:dart_api",
-    "//third_party/skia",
-  ]
+  deps = [ "//flutter/fml" ]
```

Comment:
The other deps aren't necessary?

Reply:
Correct. They were probably initially necessary but was forgotten to be removed at some point when refactoring the code.

Reply:
I wonder if some of these were copy-pasted from the macOS embedder initially? I can't imagine why we'd depend on flow, for example, given the current code. Either way, nice catch!

Reply:
They were added 6 years ago so I guess the code structure was a lot different.